lib/v2/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -44,30 +37,28 @@ class ExploreScreen extends StatelessWidget {
                   children: <Widget>[
                     Padding(
                       padding: const EdgeInsets.only(left: 16, right: 16, top: 16),
-                      child: ExploreInfoCard(
-                        onTap: () async {
-                          bool? shouldReloadExplore =
-                              await NavigationService.of(context).navigateTo(Routes.createInvite);
-                          if (shouldReloadExplore != null) {
-                            BlocProvider.of<ExploreBloc>(context)..add(const LoadExploreData());
-                          }
-                        },
-                        title: 'Invite'.i18n,
-                        amount: state.availableSeeds?.roundedQuantity,
-                        isErrorState: state.availableSeeds == null,
-                        icon: SvgPicture.asset(
-                          'assets/images/explore/person_send_invite.svg',
-                          color: AppColors.white,
-                        ),
-                        amountLabel: 'Available Seeds'.i18n,
-                      ),
-                    ),
-                    const SizedBox(height: 20),
-                    Padding(
-                      padding: const EdgeInsets.only(left: 16, right: 16),
                       child: Row(
-                        mainAxisSize: MainAxisSize.max,
                         children: [
+                          Expanded(
+                            child: ExploreInfoCard(
+                              onTap: () async {
+                                bool? shouldReloadExplore =
+                                    await NavigationService.of(context).navigateTo(Routes.createInvite);
+                                if (shouldReloadExplore != null) {
```

Comment:
shouldReloadExplore == true

if its null -> false
if its false -> false
if its true -> true

Reply:
The idea is that it works with if it returns a value that is false or true. But I agree that It can causes some confusion.